### PR TITLE
Initial version of Gem

### PR DESCRIPTION
**Pull request by alejandroechev**
This PR creates the initial version of this gem, with the following features:
 - Patch for ActiveAdmin::Filters::FormtasticAddons module
 - Patch for ActiveAdmin::Helpers::Collection module
 - Patch for ActiveResource::Connection class to support signed requests
 - ActiveAdminResource::Base class to be extended when defining ActiveResource models that need to be admined with ActiveAdmin.

All of the above are crammed together in one file for now, because of lack of knowledge from the author on how to include external files in Ruby :S 
Comments to the PR explaining how to do this and other Gem creation best practices are welcomed :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,11 +49,11 @@ class Account < ActiveAdminResource::Base
   self.site = "http://localhost/signed_api/"
 
   def self.agent_id
-    "4799c30a437de97cf4c94a0a224d51cc"
```
Comment (vanhalt) [inline README.md]: Creo que deberíamos reescribir la historia de commits y eliminar este. Por si algún día hacemos esta gema pública o incluso por uso interno.

**Review by iobaixas — CHANGES_REQUESTED** [review]
@vreyn you could put each class in a separate file under the `/lib/active_admin_resource/*` directory. Then you can require them from the main `/lib/active_admin_resource.rb` file.

File: README.md (hunk 1 above)
Comment (alejandroechev) [reply README.md]: Gracias Rafa, ocupé esta herramienta para eliminar los key de la historia https://rtyley.github.io/bfg-repo-cleaner/
Aparentemente funcionó, los reemplazo por REMOVED (aunque igual aparece aquí en este comentario el key)

**Comment by alejandroechev** [comment]
@iobaixas Thanks, I pushed a cleaned-up version with each patch / class in a separate file. For some reason this didnt work at some point for me because of some Docker cache issues and I gave up, but know it worked :)

File: lib/active_admin_resource/connection_patch.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    end
+    handle_response(result)
+  rescue Timeout::Error => e
+    raise TimeoutError.new(e.message)
+  rescue OpenSSL::SSL::SSLError => e
+    raise SSLError.new(e.message)
+  end
+end
+
+ActiveResource::Connection.prepend(ConnectionExtensions)
```
Comment (iobaixas) [inline lib/active_admin_resource/connection_patch.rb]: @vreyn i think that 'setup' code like this line should be called in the main library file or in a 'railties' file and not in the 'class file'.

Comment (alejandroechev) [reply lib/active_admin_resource/connection_patch.rb]: Ok, I moved this line to the main lib file

Comment (alejandroechev) [reply lib/active_admin_resource/connection_patch.rb]: I moved the setup code to a railtie file now.

**Review by iobaixas — APPROVED** [review]

**Pull request merged**